### PR TITLE
New version: Zettlr.Zettlr version 2.0.3

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
\r\n\rAuto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/winget-pkgs-automation)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34553)

### CODE REVIEW

File: manifests/z/Zettlr/Zettlr/2.0.3/Zettlr.Zettlr.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+PackageIdentifier: Zettlr.Zettlr
+PackageVersion: 2.0.3
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
```
Comment (OfficialEsco) [inline manifests/z/Zettlr/Zettlr/2.0.3/Zettlr.Zettlr.installer.yaml:12]:
```suggestion
UpgradeBehavior: install
ProductCode: 'd0629383-01b2-5ce9-b07a-220f17f8215a'
```

**Review by OfficialEsco — COMMENTED** [review]
Since the PackageName is dumb we can probably add the ProductCode for now?
image

**Comment by vedantmgoyal2009** [comment]
What??? I think ARP entries is a thing about which no developer cares about. Literally anything...

**Review by OfficialEsco — APPROVED** [review]
Yeeah..
I think some of them might be abit confused tho, because its a "uninstall" entry..

But you're right either way, they don't seem to check what ARP shows their software as

**Pull request merged**